src/main/java/uk/gov/companieshouse/overseasentitiesapi/model/dto/BeneficialOwnerIndividualDto.java

```diff
@@ -8,46 +8,61 @@
 
 public class BeneficialOwnerIndividualDto {
 
-    @JsonProperty("first_name")
+    public static final String FIRST_NAME_FIELD = "first_name";
+    public static final String LAST_NAME_FIELD = "last_name";
+    public static final String DATE_OF_BIRTH = "date_of_birth";
```

Comment:
Wondering why all these constants don't have _FIELD at the end of their names. We should be consistent.